stdlib/3/os/__init__.pyi

```diff
@@ -121,6 +121,8 @@ if sys.version_info >= (3, 6):
 _PathType = Union[bytes, Text]
 
 if sys.version_info >= (3, 6):
+    _PathType = Union[bytes, Text, PathLike]
```

Comment:
It's used below, and it's intended to override `_PathType` defined above (line 121). But you can't define a type alias this way, you should define in exactly once in each version-specific branch.

Also there is another definition of `_PathType` in stdlib/3/os/path.py, which you probably also want to adjust.